### PR TITLE
feat: support send message to Telegram as markdown

### PR DESCRIPTION
**Pull request by vietanhduong**
Send a message to Telegram as Markdown

### CODE REVIEW

**Comment by vietanhduong** [comment]
@ryota-sakamoto, Would you review this PR please?

File: pkg/services/telegram.go [hunk 1]
```diff
@@ -25,18 +25,27 @@ func (s telegramService) Send(notification Notification, dest Destination) error
 		return err
 	}
 
+
```
Comment (pasha-codefresh) [inline pkg/services/telegram.go:28]: Please fix linter issues

**Review by pasha-codefresh — CHANGES_REQUESTED** [review]

**Review by pasha-codefresh — APPROVED** [review]

**Review by alexmt — APPROVED** [review]

**Comment by pasha-codefresh** [comment]
Thank you @vietanhduong !

**Comment by ajax-bychenok-y** [comment]
Can somebody share an example how to use message in markdown through argo-notification? I can't manage to do so.

**Pull request merged**